src/main/java/org/jmxtrans/agent/JmxTransAgent.java

```diff
@@ -23,6 +23,7 @@
  */
 package org.jmxtrans.agent;
 
+import com.sun.jmx.defaults.JmxProperties;
```

Comment:
Should we inline the value of the constant `JmxProperties.JMX_INITIAL_BUILDER` to not have isolation issues when using Java 9?

Reply:
Good catch, didn't realize that was a com.sun class.